authui/src/authflowv2/components/tertiary-button.css

```diff
@@ -0,0 +1,78 @@
+@layer components {
+  :root {
+    --tertiary-btn-bg-color: var(--c-grey-700);
+    --tertiary-btn-text-color: var(--c-white);
+    --tertiary-btn-border-radius: var(--border-radius-full);
+    --tertiary-btn-font-family: var(--font-family-label-medium);
+    --tertiary-btn-font-size: var(--font-size-label-medium);
+    --tertiary-btn-line-height: var(--line-height-label-medium);
+    --tertiary-btn-letter-spacing: var(--letter-spacing-label-medium);
+    --tertiary-btn-font-weight: var(--font-weight-label-medium);
+    --tertiary-btn-ring-width: 0px;
+    --tertiary-btn-ring-color: transparent;
+    --tertiary-btn-py: var(--padding-2\.5);
+
+    --tertiary-btn-px: var(--padding-6);
```

Comment:
Let's calculate the paddings from the font size, just like primary-btn.

According to design, the y padding is 10, and font size is 14. So y padding can be `0.7143em` (10 / 14 = 0.71428571428). Document how we calculated it.

And x padding is half of the button height, just like primary button. This is because the maximum border radius is half of the height.

So `--padding-2\.5` and `--padding-6` could be removed.